cli/bin/setup

```diff
@@ -1,3 +1,14 @@
 #!/usr/bin/env bash
-echo "disabled until ZMQ is removed"
-# npm install --loglevel error --depth 0
+
+npm install --loglevel error --depth 0
+(cd ../exo-add && bin/setup && node_modules/.bin/build && echo 'exo-add √')
```

Comment:
The `setup` script is only to set up that particular code base, not the entire app. I like the idea of having a script that sets up the entire app, though. Let's call it `setup-all` and put it in the root directory of the monorepo! Then it can use Morula.